project_event/models/project_project.py

```diff
@@ -30,6 +30,30 @@ class Project(models.Model):
     notes = fields.Text(
         string='Notes',
     )
+    state = fields.Selection([
+        ('draft', 'Draft'),
```

Comment:
@eilst : string='Number', can be included for code field

Reply:
Done